security/fma-operator-fee.md

```diff
@@ -0,0 +1,115 @@
+  - [FM4: Database Growth Impact on Nodes](#fm4-database-growth-impact-on-nodes)
+- [Action Items](#action-items)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+|                    |                                                    |
+| ------------------ | -------------------------------------------------- |
+| Author             | leruaa                                             |
+| Created at         | 2025-01-08                                         |
+| Initial Reviewers  |                                                    |
+| Need Approval From | maurelian                                          |
+| Status             | In Review                                          |
+
+## Introduction
+
+This document covers initial deployment of Operator Fee.
+
+The OperatorFee is a new transaction fee that allows new OP chain variants to account for their unique cost structure. For example, the existing fee structure isn't friendly to chains using alt-DA's, since the l1fee only accounts for Ethereum's blobGasFee instead of an alt-DA's fee.
+
+Also, For OP Stack variants that want to utilize ZK proofs, the cost of ZK proving a transaction is a significant resource that is not taken into consideration in the current fee structure.
+
+Below are references for this project:
+
+- [Design Doc](../protocol/operator-fee.md)
+- [Spec](https://github.com/ethereum-optimism/specs/pull/382)
+
+## Failure Modes and Recovery Paths
+
+### FM1: Operator Fee scalars are set to incorrect values
+
+- **Description:** 
+  If the operator fee scalars are incorrectly initialized or updated, there is a risk that the transcations fees will be too high. This could lead to a situation where the chain become unusable.
+- **Risk Assessment:**
+  High impact, low likelihood.
+  **Mitigations:**
+  Before setting or updating the operator fee params, the operator should carefully read the [corresponding specs](https://specs.optimism.io/protocol/isthmus/exec-engine.html#operator-fee) and simulate the impact of operator fee on the whole transaction cost.
+- **Detection:** 
+  Monitor the transaction costs and alert if it's above a threshold.
+- **Recovery Path(s)**:
+  If the operator fee parameters are set to unreasonable values, the rollup operator should update the `operatorFeeScalar` and `operatorFeeConstant` to reasonable values as soon as possible.
+
+### FM2: Broken Fee Estimation (Wallets)
+
+- **Description:** 
+  If wallets fail to update their fee estimation logic, users will no longer be shown the accurate costs of a transaction.
+- **Risk Assessment:**
+  Medium impact, medium likelihood.
+  **Mitigations:**
+  Coordinate with wallet providers to update their fee estimation logic. This includes MetaMask, Coinbase Wallet, and others.
+- **Detection:** 
+  Confirm that wallets are using the correct fee estimation logic post-launch. This can be done manually on chains that have added an operator fee.
```

Comment:
> This can be done manually on chains that have added an operator fee.

Can you clarify this sentence in the FMA? I'm unsure what can be done manually